apps/web-giddh/src/app/contact/account-statement/account-statement.component.ts

```
@@ -118,17 +120,33 @@ export class AccountStatementComponent implements OnInit, OnDestroy {
+        ])
+        .pipe(takeUntil(this.destroyed$))
+        .subscribe(([response, getAccountStatementInProgress]: [any, boolean]) => {
+            let loaderTimeout = 200; // default
+    
+            // First priority: if in progress, use longer timeout
+            if (getAccountStatementInProgress) {
+                loaderTimeout = 300;
+            }
+    
+            // Second priority: if response is present, add 200ms
             if (response && response.transactionDetailList?.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)